### ISSUE TITLE
False positive unquoted resource title for colons in resource parameters

### ISSUE THREAD

**Issue by rharrison10** [rodjek/puppet-lint#146]
`puppet-lint` will falsely throw a warning for unquoted resource titles on the following manifest:

``` puppet
class test {
  exec { "${::hostname}_fix_root_pw" :
    command => '/bin/echo root:\!! | /usr/sbin/chpasswd -e',
    onlyif  => '/usr/bin/test ! `/bin/grep \'^root:!!:\' /etc/shadow`',
  }
}
```

``` bash
$ puppet-lint --no-autoloader_layout-check ./test.pp
WARNING: unquoted resource title on line 4
$ 
```
